R/single_sample_input_qc.R

```diff
@@ -263,3 +263,19 @@ sce_final_input_qc_fn <- function(sce_selected, gene_annotation) {
 
   return(sce_final_input_qc)
 }
+
+#' Return an informative message about the used operator to join cell QC filters
+#'
+#' @param operator A character scalar: used operator.
+#'
+#' @return A character scalar: the message
+#'
+#' @concept single_sample_input_qc_fn
+#' @export
+get_used_qc_filters_operator_desc <- function(operator = c("&", "|")) {
```

Comment:
Well, the AND and OR in the docs are just labels for & and |, but parameter docs explicitly list them:
https://github.com/bioinfocz/scdrake/pull/7/files#diff-447143101ad49568428bfbca1c7b9a062e2bef3e67d1ea5064c5166704f09afcR254

So I would change the docs if they are unclear, rather then implement an additional logic :smile:

Reply:
> Well, the AND and OR in the docs are just labels for & and |, but parameter docs explicitly list them: https://github.com/bioinfocz/scdrake/pull/7/files#diff-447143101ad49568428bfbca1c7b9a062e2bef3e67d1ea5064c5166704f09afcR254
> 
> So I would change the docs if they are unclear, rather then implement an additional logic 😄

Ok, in this case, disregard my comments. For clarity, I would either stick to one or the other where you mention it. I would recommend AND/OR, because it's easier to read. 
I missed the docs file 😅 The operators can stay as they are ( `&` and `|`) 👍

Once you add the changes, I will run the tests and merge and squash if everything is OK 🙂